auth/CMakeLists.txt

```diff
@@ -27,5 +27,6 @@ target_link_libraries(auth
     Seastar::seastar
     xxHash::xxhash
   PRIVATE
+    cql3
```

Comment:
The error quoted above mentions rust, not cql.

Reply:
right. but the dependency chain looks like

auth => cql3 => lang/wasm_instance_cache.hh => wasmtime_bindings (rust/cxx.h)

so auth is not obliged to link against rust. cql3 does. if we plan to add a module (yeah, i am think about the C++20 modules!) or a cmake library target of "lang", then we would need to have following dependency chain:

auth => cql3 => lang => wasmtime_bindings

and again, cql3 will be linked against lang, and lang will be linked against wasmtime_bindings.

that's why i added cql3 to the libraries linked by auth.

Reply:
Ok, makes sense.